docs/server/commands.rst

```diff
@@ -339,6 +342,25 @@ Available options:
 
   Default: ``name``.
 
+.. django-admin-option:: --only-emails
+
+  .. versionadded:: 2.8.0
+
+  Specifies to only output user names and emails. Users with no email are
+  skipped, thus anonymous users are not listed.
```

Comment:
Note there's only one anonymous user, `nobody` — leaving it as _Users with no email are skipped._ feels enough to me.